pkg/api/graph/interfaces.go

```diff
@@ -107,5 +105,5 @@ func (m ByKey) Less(i, j int) bool {
 type Suggestion string
 
 func (s Suggestion) String() string {
-	return fmt.Sprintf("try: %s", string(s))
```

Comment:
That's 100% incorrect - this is the most important info we have ever
returned in oc status, because it is what every user who tries to run
openshift with their custom image the first time hits.  If we can't explain
it here, we should scrap the project and start over.



On Thu, Jan 28, 2016 at 12:55 PM, Michail Kargakis <notifications@github.com
> wrote:

> In pkg/api/graph/interfaces.go
> <https://github.com/openshift/origin/pull/6866#discussion_r51159145>:
>
> > @@ -107,5 +105,5 @@ func (m ByKey) Less(i, j int) bool {
> >  type Suggestion string
> >
> >  func (s Suggestion) String() string {
> > -	return fmt.Sprintf("try: %s", string(s))
>
> Most of the information provided in this PR is information not directly
> useful to a user. We may need an additional field (AdditionalInfo or some
> such) in the marker that will hold that? I would vote for keeping try: to
> provide one-liners that users can cp and use.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6866/files#r51159145>.
>